community/working-groups.md

```diff
@@ -0,0 +1,44 @@
+## Working Groups
+
+A working group is a group of people that are interested in working on a common
+topic. Working groups are informal and have no official authority. All working
+groups have a public channel on the [Julia Slack][slack]. Many working groups
+also hold regular meetings that are held online and are open to anyone. Meeting
+links are posted in the relevant Slack channels.
+
+The working group meetings are also listed on the
+[community events calendar][commmunity-events-calendar].
+
+### List of working groups
+
+Note: "ET" refers to Eastern Time.
+
+| Working Group                    | Slack channel       | Meetings                                 | Contact                                                         |
+| -------------------------------- | ------------------- | ---------------------------------------- | --------------------------------------------------------------- |
+| Base Julia CI                    | `#ci-dev`           | Every other Tuesday at 2:15pm ET         | Elliot Saba (@staticfloat), Dilum Aluthge (@DilumAluthge)       |
+| GPU                              | `#gpu`              | Every other Monday at 11am ET            | Tim Besard (@maleadt), Valentin Churavy (@vchuravy)             |
+| High-performance computing (HPC) | `#hpc`              | Fourth Tuesday of each month at 2pm ET   | Valentin Churavy                                                |
+| Machine learning                 | `#machine-learning` | Every other Tuesday at 12pm ET           | Kyle Daruwalla (@darsnack), Lorenz Ohly (@lorenzoh)             |
```

Comment:
Unless you think it's too confusing. The call is always cross posted to Slack.

```suggestion
| Machine learning                 | `#machine-learning` or ([`#ml-contributors` on Zulip](https://julialang.zulipchat.com/#narrow/stream/237432-ml-contributers) | Every other Tuesday at 12pm ET           | Kyle Daruwalla (@darsnack), Lorenz Ohly (@lorenzoh)             |
```

Reply:
I'm not sure if there's enough real estate in the table itself, but I can put a note under the table.

Reply:
I've put it in a note below the table. However, if we get to the point where multiple WGs are using Zulip, we could consider adding a Zulip column to the table.